zh-tw.json: update translations to current english head

medicalwei opened the pull request:
Disclosure: part of the contents in Settings Description and Help contents are assisted by generative AI.  Manual review has been taken in-world.

Gawdl3y commented:
Hi, thanks for your contribution! Just to clarify, what aspect of the translations was generative AI involved with? Were its edits/additions manually reviewed?

medicalwei commented:
I am taking some of the suggestions as these seems fitting:

> ```
> 目前在私人世界（共開啟 %total_worlds% 個世界）
> 與其他 %current_world_users% 人在 %current_world_name%（共開啟 %total_worlds% 個世界）
> a bit ungrammatical (the simp chin trans also suffers from that issue but also simp chin translation of computer interface text has always been heavily ungrammatical- zh-tw does not do that though)
> i would say:
> 處於私人世界中（共有載入[1] %total_worlds% 個世界）
> 與其他 %current_world_users% 位用戶[2]處於 '%current_world_name%'[3] 中（共有載入 %total_worlds% 個世界）
> [1] 開啟 is a *valid* way to say "open" as in "open a new page", but not in the sense of "having something open", 載入 means load/loaded, compare 「共開啟10個程式」 vs 「共載入10個程式」 ("has 10 programs open"). and also 有 is needed to convey "this is the total loaded worlds *so far*", it's like the present perfect tense in English (i dont know if this is connected i.e. if this usage of 有 is translated into modern chinese, lmao)
> [2] it says users in the english text, so we should respect that specificity and use 用戶(it's the standard computer UI translation for 'users'), and also you need to say 位 or 個 because `與其他 5 人` is not grammatical
> [3] English *names* or proper nouns in a Chinese UI need to be quoted, usually single quotes. this is because `與其他 5 位用戶處於 MyEpicGrid 中` is just.. awkward, compared to `與其他 5 位用戶處於 'MyEpicGrid' 中`

1. I don't see 有 is needed in this sentence, also 共有 can be a confusing term like "shared" as in "shared resources".  I am also not sure about 開啟 vs 載入, although I agree that "Opening" a "world" sounds weird to people not in context.  I would prefer "enter" / "in", but the original text uses "open" so I just translate it as-is.
2. Users in Taiwanese Chinese (idk about zh-hk) mostly uses 使用者 although 用戶 is also common.
3. Referring to other non-latin translations, it is not quoted, also, names of the worlds may not be in English (for example ケモノ広場), and I think using simple space should be sufficient to tell apart.

> "General.OK": "OK",
> 「好的」 is a pretty standard translation, "OK" is anglocentrism ('Ok' is not remotely widespread amongst chinese speakers)

"OK" is a difficult term to convey (see [電腦介面中的「OK」](https://fredjame.com/%E9%9B%BB%E8%85%A6%E4%BB%8B%E9%9D%A2%E4%B8%AD%E7%9A%84-ok-2af0eaf59416)), as it may not only be something for user to confirm, it may also be something for user to start a function from the modal dialog.

> "General.Finish": "結束",
> i can't check this one in context (please check it in context! i am not a resonite translator so i have no idea where these texts can be found in game) but i think 「完成」 is more fitting?

I am not sure where this text would appear, but there is also Done (完成！) in this translation.

> "General.Reset": "重置",
> depending on the context (i.e. if it's meant to be used as in "reset config?"), 重設 might be more fitting

Changes done

> 確定要開啟嗎？
> see the previous point, say 載入 instead (again, the thing here is about *nuance*, aka i am nitpicking the usage of 開啟 here)
> 
> "General.Saved": "儲存完成！"
> `儲存完畢！`

Changes done.

> 
> "General.ClearSelection": "清除選擇",
> idk what context this is in but if it's e.g. "clear selected options" or "clear selected filters" it should be 清除選單, and if it's "clear selected text" or "clear selected objects" it should be 取消選取

Changes done.

> 
> "General.Off": "關閉",
> "General.On": "打開",
> idk the context of these but those are 99% not the correct translations

I am not sure about the context neither. 

> 
> "General.Understand": "我明白了",
> `了解`

I put it as "我瞭解了"

> 
> "General.Notice": "注意",
> it should be either `注意事項` or `請注意：` depending on the context

I am unsure about the context, but I changed it to 通知 and see whether it works fine.

> 
> "General.UnderstandEnable": "我明白了，請啟用",
> the punctuation should be 。 not ，, and depending on the context 啟用 might not be correct
> 

English comma and full stop are used differently vs. Chinese ones.  Chinese sentences do not need to be a full sentence to use a full stop, and for me it seems to be awkward to use a 。 here.

> "General.UI.SearchPromptLong": "輸入以搜尋…",
> that looks ungrammatical but idk the context so i cant know what it should be

That's used in the world search menu. I changed it to "在此輸入文字以搜尋……"

> 
> "General.UI.Subtitles.On": "字幕：打開",
> "General.UI.Subtitles.Off": "字幕：關閉",
> `顯示字幕` and `隱藏字幕`. "something": "some state" is anglocentrism and should be minimised outside of technical contexts (e.g. in a UI) (notable exception being the volume controls just downstairs).

This seems to be inside a video player, unsure whether the actual context is.  Similar usage includes the user scaling and laser enable/disable in the context menu, and it seems to represent current state instead of the state when you press the option.

> 
> "General.UI.PressToBegin": "按下以開始",
> ungrammatical, idk the context but `按下開始` would be more grammatical in any contexts anyway
> 

I didn't changed it because 按下開始 sounds too weird as a button.

> "Worlds.Home": "自家",
> `我的世界` is unironically the better translation here ('自家' has a lot of unnecessary implications) especially if it'd be put into uhh `回到我的世界`, but context is still important
> 

I changed it to 我的家園 as 我的世界 (my worlds) can be many but user can have one home set as a preferred world.

I haven't changed the locomotion part yet,

Frooxius commented:
Thank you for the translations! Merging in!
